Sources/SwiftTwitch/Classes/Results/Videos/VideoData.swift

```diff
@@ -81,25 +81,4 @@ public struct VideoData: Unmarshaling {
     /// `durationString` defines the duration of the video as a String in a `HH'h'mm'm'ss's' format.
     public let durationString: String
 
-    /// Initializes a `VideoData` object from the input `MarshaledObject`. This will throw if there
-    /// is missing data from the input `MarshaledObject`.
-    ///
-    /// - Parameter object: The object to initialize a `VideoData` piece from
-    /// - Throws: If data is missing that was expected to be non-`nil`.
-    public init(object: MarshaledObject) throws {
-        id = try object.value(for: Twitch.WebRequestKeys.id)
-        ownerId = try object.value(for: Twitch.WebRequestKeys.userId)
-        ownerName = try object.value(for: Twitch.WebRequestKeys.userName)
-        title = try object.value(for: Twitch.WebRequestKeys.title)
-        description = try object.value(for: Twitch.WebRequestKeys.description)
-        creationDate = try object.value(for: Twitch.WebRequestKeys.createdAt)
-        publishedDate = try object.value(for: Twitch.WebRequestKeys.publishedAt)
-        url = try object.value(for: Twitch.WebRequestKeys.url)
-        thumbnailURLString = try object.value(for: Twitch.WebRequestKeys.thumbnailURL)
-        viewSetting = try object.value(for: Twitch.WebRequestKeys.viewable)
```

Comment:
Different property name vs web request key name, need to change variable name or do custom Codable initialization